scripts/jest-enzyme-setup.js

```diff
@@ -3,7 +3,7 @@
  * Jest will run this file before executing each test file.
  */
 import Enzyme from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
Oof, I'm not a huge fan of using anything unofficial but since there hasn't been much movement on https://github.com/enzymejs/enzyme/issues/2429, I'm okay with this for now (especially since it only affects a small number of unit tests).

[Looks like something official may come soon](https://github.com/enzymejs/enzyme/issues/2429#issuecomment-763078234).

Reply:
True, that's the plan. Once something official comes along we will switch, given we still have enzyme usage in our code base.

Reply:
>that's the plan. 

Ticket!

Reply:
Yes, and we're planning to have @jcalcaben help rewrite our Enzyme tests anyway. Really want to get rid of that library.

Reply:
> Ticket!

https://jira.corp.magento.com/browse/PWA-1588